server/core/include/irods/irods_notify_service_manager.hpp

```diff
@@ -0,0 +1,56 @@
+#ifndef IRODS_NOTIFY_SERVICE_MANAGER_HPP
+#define IRODS_NOTIFY_SERVICE_MANAGER_HPP
+
+#include <fmt/format.h>
+#include <fmt/compile.h>
```

Comment:
Unless the include order was intentional...
```suggestion
#include <fmt/compile.h>
#include <fmt/format.h>
```

Reply:
Done